src/main/java/InputHandler.java

```diff
@@ -1,9 +1,14 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
 public class InputHandler {
 
     public static Command parse(String input) throws DukeException {
 
         String[] processedInput = input.split(" ");
         String command = processedInput[0];
+        String description = processDescription(processedInput);
```

Comment:
I noticed that you often used the work "process..." when naming your functions. I think process is kinda vague and may confuse the reader. Maybe you can consider a change like convertDescriptionStringTo... instead of processDescription